### ISSUE TITLE
Character creation menu tabs bug

### ISSUE THREAD

**Issue by Ryan-Blackwell** [NovaSector/NovaSector#435]
Reporting client version: 514.1589

## Issue Summary

Within the character creation menu, the underclothes menu appears on the wrong side of the portrait, behind the image, and the "Job/Loadout/Underwear/Naked" drop-down beneath it is pushed down too far to select the lower options, blocking me from changing character details and seeing the full model pre-game. I can pull the menu window down to get the clothing preview changed, but I cannot alter it in a way that lets me see the underclothes menus.

![image](https://github.com/NovaSector/NovaSector/assets/107783314/a919a745-f730-416a-b12e-be482916aa96)
![image](https://github.com/NovaSector/NovaSector/assets/107783314/a3229382-4a6e-4274-a1d6-68460526861d)

## Round ID:
10552

## Testmerges:
- [Antag optin ](https://github.com/NovaSector/NovaSector/pull/121)
- [[TM ONLY] [EARLY MIRROR] Revert "Makes immerse use weakrefs", prioritizing some ref removal lines instead](https://github.com/NovaSector/NovaSector/pull/238)
- [[MODULAR] Removes Arbitrary Loadout Restrictions](https://github.com/NovaSector/NovaSector/pull/58)
- [[TM-ONLY] Testing chatmessage debug](https://github.com/NovaSector/NovaSector/pull/275)
- [[MIRROR] Replace natural beheading with cranial fissures (splitting your skull) (#80703)](https://github.com/NovaSector/NovaSector/pull/382)
- [The Thrill of the Hunt: Hemophage blood digestion changes](https://github.com/NovaSector/NovaSector/pull/93)
- [[Semi-Modular]Ports over the old Food Perfs PR](https://github.com/NovaSector/NovaSector/pull/352)

**Comment by SomeRandomOwl** [comment]
@vinylspiders I dont think this is related to my pr i have been noticing some jank with those menus before the tm too such as the tooltips for each item rendering under the window but let me check my local to make sure its not my pr
